src/main/java/nl/hsac/fitnesse/util/iban/IbanUtil.java

```diff
@@ -0,0 +1,129 @@
+package nl.hsac.fitnesse.util.iban;
+
+import nl.hsac.fitnesse.util.RandomUtil;
+
+
+/**
+ * To select a country and parse the iban request to the proper class
+ */
+public class IbanUtil {
+    private final RandomUtil RANDOM_UTIL = new RandomUtil();
+
+
+    public String generateIban(String country, String bankCode) {
+        String iban;
+        if (country.equals("")) {
+            country = RANDOM_UTIL.randomElement(countryCodes);
+        }
+
+        switch (country) {
+            case "BE":
+                BEIban beIban = new BEIban();
```

Comment:
Can you make all the xxIban classes extend a single superclass, or implement an interface, so this switch statement becomes responsible only for selecting the class and the actual generation (using a `generate` method defined by the superclass/interface) and return statements can be moved outside the switch (i.e. are shared)?